### DISCUSSION TITLE
Release v0.6 is ready!

### DISCUSSION THREAD

**Discussion by keithschacht** [AllYourBot/hostedgpt#305] [Announcements]
I officially wrapped up the latest release. It should be stable and the app is starting to work really well. I previously announced support for markdown, since then I added the ability to abort a long reply that's coming in (by clicking the stop icon or simply interrupting with a new question), the ability to edit your messages (you can browse versions with left/right arrows), and a first pass at mobile PWA support (thanks Jason!). Plus, lots of continual polish and bug fixes.

I made a short video highlighting the key features:
[![](https://p425.p0.n0.cdn.zight.com/items/qGubwRKr/c4a119a9-254d-454a-b602-610b428ee769.jpg)](https://www.youtube.com/watch?v=hXpNEz-slkU)

I just shared this on reddit so you can help me out by chiming in there!
https://www.reddit.com/r/rails/comments/1cdxjb3/a_new_rails_hostedgpt_is_out_what_should_come_next/

cc: @jasonpaulso @stephan-buckmaster @anandvc @florianfelsing @olimart @robacarp @marckohlbrugge @al3rez @Tabenkin @matthewbennink

**Comment by stephan-buckmaster** [comment]
I get an error when starting up the server in development mode, any tips? I was going to increase the limit in the number of open files.

```
Rebuilding...
node:internal/errors:465
    ErrorCaptureStackTrace(err);
    ^

Error: EMFILE: too many open files, watch '...../app/assets/stylesheets/application.tailwind.css

  at createFsWatchInstance (/snapshot/tailwindcss/node_modules/chokidar/lib/nodefs-handler.js:119:15)
: [stacktrace]
:
Detected tailwind has gone away, stopping Puma...
- Gracefully stopping, waiting for requests to finish
```

**Reply by stephan-buckmaster** [reply]
Simply waiting resolved this.

**Reply by keithschacht** [reply]
Hmm, this one is new to me. I'm glad it resolved itself, but it would be nice for us to get the root cause fixed.

Were you starting things up using docker or just `bin/dev`?
